src/api/v1/permissions.json

```diff
@@ -116,6 +116,7 @@
     "VOLUNTEER_ADMIN": [
       "constants-all:read",
       "organisations_outreach-child:read",
```

Comment:
Some more strange permissions here and directly below. Would have expected `organisations_outreach-own:read` instead. No action in this PR, I'm just noting so I can make an issue.